ssl/ssl_key_share.cc

```diff
@@ -203,11 +203,16 @@ class CECPQ2KeyShare : public SSLKeyShare {
     uint8_t x25519_public_key[32];
     X25519_keypair(x25519_public_key, x25519_private_key_);
 
-    uint8_t hrss_entropy[HRSS_GENERATE_KEY_BYTES];
+    Array<uint8_t> hrss_entropy;
+    if (!hrss_entropy.Init(HRSS_GENERATE_KEY_BYTES)) {
+      OPENSSL_PUT_ERROR(SSL, ERR_R_MALLOC_FAILURE);
+      return false;
+    }
+
     HRSS_public_key hrss_public_key;
-    RAND_bytes(hrss_entropy, sizeof(hrss_entropy));
+    RAND_bytes(hrss_entropy.data(), hrss_entropy.size());
     if (!HRSS_generate_key(&hrss_public_key, &hrss_private_key_,
-                           hrss_entropy)) {
+                           hrss_entropy.data())) {
       return false;
     }
```

Comment:
Did you intend to include this change in this PR?  It seems unrelated to zeroize'ing the ED25519 seed.

Reply:
Yes, I repurposed this PR :)